src/fracas/syntax-regex.ts

```diff
@@ -48,6 +48,10 @@ export function anyDefineSymbolRx(symbol: string, searchKind = SearchKind.wholeM
         `(?<=[${RX_CHARS_OPEN_PAREN}])\\s*(${RX_SYMBOLS_DEFINE})\\s*[${RX_CHARS_OPEN_PAREN}]?\\s*(${escapeForRegEx(symbol)}${RX_CHAR_IDENTIFIER}*)(?!${RX_CHARS_CLOSE_PAREN})`;
 }
 
+export function anyKeySymbolRx(symbol: string): string {
+    return `([${RX_CHARS_OPEN_PAREN}])\\s*(define-key)\\s+${escapeForRegEx(symbol)}(${RX_CHARS_CLOSE_PAREN})`;
```

Comment:
https://regex101.com/r/j5bQZn/1

In case you want to play around with it

Reply:
btw general Racket/Scheme/Lisp tooling gave up on this long ago and just considers anything starting with `define` (or `def` in Common Lisp's case) to be a definition worth cataloging (and also worth formatting the same way). So, another way to do this is to adjust `RX_SYMBOLS_DEFINE` to take anything starting with `define`.

Reply:
Strange ... `define-key` is already part of `RX_SYMBOLS_DEFINE`, but it's not resolving properly. @jm991 your change shouldn't even be necessary...

Reply:
@thisisthedave I think it's because the regex was expecting there to be something following the `(define-key {symbol})` pattern - which is not the case (this is an exception to the pattern for all other `define`s). Eg.:
```
(define-key *my-key*)
```
vs.
```
(define *my-key* 'my-key)
```

Reply:
I found the bug with this. I added a check that the definition isn't immediately closed to filter out forward declarations, but `(define-key whatever)` looks like a forward declaration. I'll fix the regex, which will make this PR obsolete. (Appreciate the thought though 😬)

Reply:
k! lemme know when the change is in `vscode-fracas/master` and I'll give it a spin and close out this PR as obsolete.